auto h-encore/Language.cs

```diff
@@ -967,6 +967,99 @@ public static class Language {
+                { "error_InvalidOperationException",  "Nem jó a letöltés. Ellenőrizd hogy stabil e az interneted."},
+                { "error_TargetInvocationException", "MD5 HIBA." },
+                { "error_Template", "Hiba {0} történt.\r\n\r\n{1}\r\n\r\nKérlek próbáld újra. If you can't solve the issue, please create an issue on the issue tracker with this error code." },
+                { "error_Redownload",  "Hiba 1001-0105\r\n\r\nNem sikerült letölteni a fájlt {0}\r\n\r\nEllenőrizd az interneted és/vagy próbáld újra. ha nem működik, create an issue on the Github issue tracker."},
+
+                { "log_SearchingForQCMA", "QCMA keresése..." },
+                { "log_FoundQCMA", "QCMA találva." },
+                { "log_QCMANotFound", "QCMA nem található, letöltés." },
+                { "log_KillingQCMA", "futó QCMA alkalmazás kinyírása..." },
+                { "log_QCMARegistry", "Importing QCMA registry information..." },
+                { "log_ScrubAID", "Scrubbing AID value" },
+                { "log_Prompt", "infoormáció megadása..." },
+                { "log_Done", "        kész!" },
+                { "log_WipeFiles", "Régi fájlok törlése..." },
+                { "log_Import", "Fájl importálés ennek a fájlnak {0} rendben." },
+                { "log_DownloadValid", "Fájl {0} már levan töltve és rendben van, nem kell újratölteni." },
+                { "log_DownloadInvalid", "Fájl {0} levan töltve de szar, újra lesz töltve." },
+                { "log_NotDownloaded", "Fájl {0} nincs letöltve sem importálva, lefogja tölteni." },
+                { "log_WorkingDirs", "Könyvtárak Generálása..." },
+                { "log_CorrectLocation", "Fájl {0} jó helyen van, átugrás." },
+                { "log_Importing", "Importálás {0}" },
+                { "log_Downloading", "Letöltés {0}" },
+                { "log_Extracting", "Kibontás {0}" },
+                { "log_ExtractingPKG", "bittersmile demó kibontása..." },
+                { "log_Trimming", "Felesleges fájlok kiírtása a bittersmile demóból..." },
+                { "log_MoveToHencore", "Mozgatás {0} a h-encore mappájába..." },
+                { "log_MoveLicense", "License fájl mozgatása..." },
+                { "log_GetCMA", "CMA titkosító kulcs megszerzése AID al {0}" },
+                { "log_GotCMA", "CMA titkosító kulcs megszerezve {0}" },
+                { "log_Packaging", "h-encore csomagolása {0} psvimgtools al..." },
+                { "log_MoveToQCMA", "h-encore fájlok mozgatása a QCMA APP környvtárába...\r\n" },
+                { "log_Finished", "auto h-encore Végzett!!\r\n" },
+
+                { "title_Main", "auto h-encore" },
+                { "title_Import", "Létező fájlok importásása" },
+                { "title_Warning", "Figyelmeztetés" },
+                { "title_Error", "Hiba" }
```

Comment:
} }

Reply:
@noahc3  Thanks.
how soon may i expect the builded version with the translation?

Reply:
I'm going to add 32bit support to h-encore then bundle it with 0.4.4. Soon.